core/markets/storage/stored_ask.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "markets/storage/stored_ask.hpp"
+
+namespace fc::markets::storage {
+
+  outcome::result<void> StoredAsk::addAsk(const TokenAmount &price,
+                                          ChainEpoch duration) {
+    std::lock_guard<std::mutex> lock(mutex_);
+    uint64_t sequential_number{0};
+    if (signed_storage_ask_) {
+      sequential_number = signed_storage_ask_->ask.seq_no + 1;
+    }
```

Comment:
Move to `ask.seq_no = signed_storage_ask_ ? signed_storage_ask_->ask.seq_no + 1 : 0;`